fmriprep/cli/run.py

```diff
@@ -76,6 +77,11 @@ def get_parser():
     # Re-enable when option is actually implemented
     # g_bids.add_argument('-r', '--run-id', action='store', default='single_run',
     #                     help='select a specific run to be processed')
+    g_bids.add_argument(
+        '--bids-filters', action='store', type=Path,
```

Comment:
I think making it explicit this is a file will improve clarity and be more in line with the other file cli arguments
```suggestion
        '--bids-filter-file', action='store', type=Path,
```